Dockerfile

```diff
@@ -4,8 +4,9 @@ WORKDIR /root
 # Installs sndfile library for reading and writing audio files 
 RUN apt-get update && apt-get install -y libsndfile-dev
 
-# Instals ddsp library and upgrade tensorflow
-RUN pip install --upgrade pip && pip install --upgrade tensorflow && pip install --upgrade ddsp
+# Instals Magenta DDSP library and upgrade Tensorflow
```

Comment:
In scripts or function headers we would normally use the present tense. Like "Installs" and "upgrades".

Reply:
Thank you for pointing this out. I've changed it.